src/pages/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -21,7 +21,7 @@ The developer preview version provides a preview of the upcoming changes to the
 
 ## [v3.6] - Developer preview
 
-No changes yet.
+- Removed **List Customers** method.
```

Comment:
Hello :) Can we please do it the way it is done [here](https://api-11763-remove-list-customers-v36--livechat-public-docs.netlify.app/messaging/agent-chat-api/changelog#status) - I would add a heading - `Customers` in this case and form the sentence the way I offered in the second comment :) Thanks to this, we will have everything in order from the start :)